### ISSUE TITLE
next/font: Module '"next/font/google"' has no exported member

### ISSUE THREAD

**Issue by Juneezee** [vercel/next.js#50917]
### Verify canary release

- [X] I verified that the issue exists in the latest Next.js canary release

### Provide environment information

```bash
Operating System:
      Platform: linux
      Arch: x64
      Version: #1 SMP PREEMPT_DYNAMIC Mon, 05 Jun 2023 15:12:57 +0000
    Binaries:
      Node: 18.16.0
      npm: 9.5.1
      Yarn: 1.22.19
      pnpm: 8.6.1
    Relevant packages:
      next: 13.4.5-canary.7
      eslint-config-next: 13.4.5-canary.7
      react: 18.2.0
      react-dom: 18.2.0
      typescript: 5.1.3
```

### Which area(s) of Next.js are affected? (leave empty if unsure)

Font optimization (next/font)

### Link to the code that reproduces this issue or a replay of the bug

https://codesandbox.io/p/sandbox/delicate-monad-rfv1p1

### To Reproduce

1. `npx create-next-app@13.4.4`
2. Update `next` in `package.json` to `13.4.5-canary.7`
3. Run `npm install && npm run build`

```
npm run build                                                                                                                ✘ INT nnn 1

> my-app@0.1.0 build
> next build

Failed to compile.

app/layout.tsx
An error occured in `next/font`.

Error: Cannot find module '../dist/google/loader'
Require stack:
- /home/jun/Desktop/my-app/node_modules/next/dist/compiled/@next/font/google/loader.js
- /home/jun/Desktop/my-app/node_modules/next/dist/build/webpack/loaders/next-font-loader/index.js
- /home/jun/Desktop/my-app/node_modules/next/dist/compiled/loader-runner/LoaderRunner.js
- /home/jun/Desktop/my-app/node_modules/next/dist/compiled/webpack/bundle5.js
- /home/jun/Desktop/my-app/node_modules/next/dist/compiled/webpack/webpack.js
- /home/jun/Desktop/my-app/node_modules/next/dist/server/config-utils.js
- /home/jun/Desktop/my-app/node_modules/next/dist/server/config.js
- /home/jun/Desktop/my-app/node_modules/next/dist/build/index.js
- /home/jun/Desktop/my-app/node_modules/next/dist/cli/next-build.js
- /home/jun/Desktop/my-app/node_modules/next/dist/lib/commands.js
- /home/jun/Desktop/my-app/node_modules/next/dist/bin/next
    at Module._resolveFilename (node:internal/modules/cjs/loader:1075:15)
    at /home/jun/Desktop/my-app/node_modules/next/dist/server/require-hook.js:188:36
    at Module._load (node:internal/modules/cjs/loader:920:27)
    at Module.require (node:internal/modules/cjs/loader:1141:19)
    at require (node:internal/modules/cjs/helpers:110:18)
    at Object.<anonymous> (/home/jun/Desktop/github/my-app/node_modules/next/dist/compiled/@next/font/google/loader.js:1:18)
    at Module._compile (node:internal/modules/cjs/loader:1254:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1308:10)
    at Module.load (node:internal/modules/cjs/loader:1117:32)
    at Module._load (node:internal/modules/cjs/loader:958:12)


> Build failed because of webpack errors
```

### Describe the Bug

Unable to import font from `next/font` in `13.4.5-canary.7`.

### Expected Behavior

Able to import font from `next/font`

**Comment by lucgagan** [comment]
This is a regression in `13.4.5-canary.7`. It works fine if you downgrade back to `13.4.5-canary.6`.

**Comment by ElvarP** [comment]
Also having the same problem. Using Next Canary.

**Comment by lucgagan** [comment]
Serious question: How something like this not caught by the automated tests?

**Comment by joacub** [comment]
> Serious question: How something like this not caught by the automated tests?

Seems like they did lately some changes in their workflow  as per the commits days, so maybe a mistake …

**Comment by Juneezee** [comment]
It appears to be fixed in `13.4.5-canary.8`.

**Comment by Brettkgamble** [comment]
Still active in current canary 13.4.5-canary.11
Works in 13.4.5-canary.6
